ui/components/app/assets/asset-list/asset-list-control-bar/asset-list-control-bar.tsx

```diff
@@ -54,13 +54,13 @@ const AssetListControlBar = ({ showTokensLinks }: AssetListControlBarProps) => {
   });
 
   const allNetworksFilterShown =
-    Object.keys(tokenNetworkFilter).length !== Object.keys(allOpts).length;
+    Object.keys(tokenNetworkFilter || {}).length !== Object.keys(allOpts || {}).length;
```

Comment:
Needed to add this because the extension bricks in test mode because `tokenNetworkFilter ` is undefined.